src/gap.c

```diff
@@ -900,6 +900,40 @@ Obj FuncNanosecondsSinceEpoch(Obj self)
 
 /****************************************************************************
 **
+*F  FuncNanosecondsSinceEpochInfo( <self> )
+**
+**  'FuncNanosecondsSinceEpochInformation' returns a plain record
+**  contains information about the timers used for FuncNanosecondsSinceEpoch
+**
+*/
+Obj FuncNanosecondsSinceEpochInfo(Obj self)
+{
+  Obj res, tmp;
+  Int resolution;
```

Comment:
This should be Int8 (and in the method as well), as the resolution could fill a 32-bit integer I think